app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,15 @@
+class SessionsController < ApplicationController
+  def new
+  end
+  def create
```

Comment:
Assignment Branch Condition size for create is too high. [15.07/15]<br>Use empty lines between defs.